examples/adjustwindow_updateviewer_saveimage.py

```diff
@@ -0,0 +1,71 @@
+from napari.settings import get_settings
+import time
+import napari
+from napari._qt.qthreading import thread_worker
+from skimage import data
+from skimage.morphology import ball, octahedron
+import matplotlib.pyplot as plt
+
+
+def make_screenshot(viewer):
+    img = viewer.screenshot(canvas_only=True, flash=False)
+    plt.imshow(img)
+    plt.axis("off")
+    plt.show()
+
+
+#get_settings().application.window_size = (600, 600) # I will delete this line before merge
+get_settings().application.window_position = (900, 300) # hopefully better option to do this
+viewer = napari.Viewer()
+viewer.window.resize(600, 600) 
+#viewer.window.position(600, 600) # is it possible that this will replace get_settings().application.window_position = (900, 300)?
```

Comment:
@tlambert03 :
I've now replaced `get_settings().application.window_size` with `viewer.window.resize(600, 600)`.
Is there something similar for `get_settings().application.window_position = (900, 300)` ?

Reply:
And while I'm on it: Is there also an option in `application.window_position` to define the screen on which napari will open? That would be incredibly useful for a two-screen setup: One screen with the editor, and the viewer would show on the second screen.

Reply:
we don't have a public API for that...  you'd have to use the Qt API directly.  I don't have a second monitor to test it out

Reply:
My three cents what would be lovely to have in the user API:
* An option to resize the window: `viewer.window.resize(600, 600) `  (already there 🎉)
* An option to position the window, e.g. `viewer.window.position(600, 600)` (currently possible with `get_settings().application.window_position = (900, 300)`)
* An option to select the screen where napari will pop up, this would be amazing to have in a multiscreen setup. (I've made an issue about that idea here: https://github.com/napari/napari/issues/3987)

Reply:
I like all three! Monitor selection is tricky because I think that is very platform-specific. For example, in Linux using i3, everything appears as one giant, irregular canvas. So I think that will take a while to get right...